base/src/types.rs

```diff
@@ -110,7 +111,7 @@ pub struct Worksheet {
     pub sheet_id: u32,
     pub state: SheetState,
     pub color: Option<String>,
-    pub merge_cells: Vec<String>,
+    pub merge_cells: Vec<MergedRange>,
```

Comment:
I think I prefer `merged_range` and `merged_ranges` otherwise you would have have to go for `merged_cells` and `merged_cells_list` or something.

I am happy o discuss here, I'm not a native English speaker!

I see the point in calling a "merged cell" a cell that is it composed of many cells.

I think if we go with merged ranges we sidestep the problem completely. In that case it would be:

```rust
pub merged_ranges: Vec<MergedRange>
```

what do you think?